forms/aws/lambda/reliability/reliability.js

```diff
@@ -9,45 +9,47 @@ exports.handler = async function (event) {
   let submissionIDPlaceholder = "";
 
   const message = JSON.parse(event.Records[0].body);
-  return await getSubmission(message)
-    .then((messageData) => ({
+  try{
+    const messageData = await getSubmission(message)
+    const processedMessageData = {
```

Comment:
Just semantics but instead of creating an object that holds the values to only destructure the object in the next few lines I'm wondering if it would be easier to just initiate the different const variables straight from messageData.
Thinking..
const formID = messageData.Item ....
const formSubmission = messageData.Item ....
...